rasa/cli/visualize.py

```diff
@@ -4,11 +4,6 @@
 from typing import List
 
 from rasa import data
-from rasa.cli.arguments.default_arguments import (
-    add_config_param,
-    add_domain_param,
-    add_stories_param,
-)
 from rasa.cli.arguments import show as arguments
```

Comment:
this probably needs updating -- `rasa/cli/arguments/show` should be renamed like `rasa/cli/show` was

Reply:
good catch 👍